services/checks/checks.go

```diff
@@ -59,7 +56,6 @@ const (
 	envResendInterval  = "PERCONA_TEST_CHECKS_RESEND_INTERVAL"
 
 	checksTimeout       = time.Hour
```

Comment:
Not directly related to this PR, but… how we have one hour timeout for checks downloading and execution? O_o Let's make it smaller, rename, and add a comment

Reply:
I changed the value to 5 minutes and added a comment, but the variable name is ok for me and I haven't an idea how to name it better.